py/server/tests/test_table_data_service.py

```diff
@@ -346,8 +346,8 @@ def test_partition_sub_failure(self):
         backend.sub_new_partition_fail_test = True
         table = data_service.make_table(TableKeyImpl("test"), refreshing=True)
         with self.assertRaises(Exception) as cm:
-            # failure_cb will be called in the background thread after 2 PUG cycles
-            self.wait_ticking_table_update(table, 600, 2)
+            # failure_cb will be called in the background thread after 2 PUG cycles, 3 seconds timeout should be enough
+            self.wait_ticking_table_update(table, 600, 3)
```

Comment:
If the machine is loaded should we really say only 3 seconds?  What do we have to lose by making the value higher?  Will a higher value actually make a successful test take longer?

Reply:
A higher number does make the test take longer b/c the `awaitUpdate` method called underneath won't error out due to the table being in a failed state, so we'll exhaust the timeout.